### PR TITLE
Fix(Project UI): Removed 'Unknown' from Project Clearing Team dropdown

### PR DESCRIPTION
**Pull request by akapti**
**Summary:** removed `Unknown` option from the Project Clearing Team drop down.

**Steps to test:**
```
- Login to SW360, and open project portal

- Edit any Project listed under Project tab.

- Go to 'Administration' tab, and check the "Clearing Team" drop down.

- You should not see 'Unknown' option in drop down.
```

closes: #563 

Signed-off-by: Abdul Kapti <abdul.mannankapti@siemens.com>

### CODE REVIEW

**Comment by maxhbr** [comment]
see discussion in #563

**Comment by mcjaeger** [comment]
(from discussion) more time required to reason about it

**Comment by mcjaeger** [comment]
(from discussion) proposal is to make it configurable, whether unknown shall be shown or not.

**Comment by akapti** [comment]
made the feature configurable via sw360 properties.

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java [hunk 1]
```diff
@@ -37,7 +37,8 @@
     public static final Set<String> SET_CLEARING_TEAMS_STRING;
     public static final String LICENSE_IDENTIFIERS;
     public static final String PREFERRED_COUNTRY_CODES;
-    public static final Boolean MAINLINE_STATE_ENABLED_FOR_USER;
+    public static final boolean MAINLINE_STATE_ENABLED_FOR_USER;
+    public static final boolean IS_CLEARING_TEAM_UNKNOWN_ENABLED;
```
Comment (nutanv1contr) [inline frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java]: Please revert the change for `MAINLINE_STATE_ENABLED_FOR_USER` as the change is not related. Also change the type of variable to `Boolean`  wrapper class.

Comment (akapti) [reply frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java]: Done.

File: frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java [hunk 2]
```diff
@@ -512,6 +513,7 @@
         PROJECTIMPORT_HOSTS = props.getProperty("projectimport.hosts", "");
         PREFERRED_COUNTRY_CODES = props.getProperty("preferred.country.codes", "DE,AT,CH,US");
         MAINLINE_STATE_ENABLED_FOR_USER = Boolean.parseBoolean(props.getProperty("mainline.state.enabled.for.user", "false"));
+        IS_CLEARING_TEAM_UNKNOWN_ENABLED = Boolean.parseBoolean(props.getProperty("clearing.team.unknown.enabled", "false"));
```
[the remark this answers is not in the document]
Comment (akapti) [reply frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java]: changes the default value to `true`.

**Comment by nutanv1contr** [comment]
Tested the PR. Successfully. `unknown`  option is now configurable as per the property in sw360.properties file. Code looks good.
